tmail-backend/jmap/extensions/src/main/java/com/linagora/tmail/james/jmap/contact/jCardObject.java

```diff
@@ -0,0 +1,29 @@
+package com.linagora.tmail.james.jmap.contact;
+
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+
+@JsonDeserialize(using = jCardObjectDeserializer.class)
+public record jCardObject(String fn, String email) {
```

Comment:
```suggestion
public record jCardObject(String fn, MailAddress email) {
```

Let's use strong types!